### PR TITLE
Front end cleanup

### PR DESCRIPTION
**Pull request by BCerki**
https://github.com/bcgov/cas-registration/issues/1183

This is PR 1/2 for this card. This PR only moves files and updates imports

In this refactor:
- cleaned up components/folders that weren't being used
- All files in /routes called Page.tsx
- Made component folders for files that were previously in /routes but that aren't actually routes or pages (e.g. the buttons to accept/decline an operation are just a component)
- I'm still not convinced that the /routes folder is the way to go. We'll have to update it anytime we change our routes, and in some cases we have the same route (users) for different user roles and the different roles need to see different pages at the same route.

PR #2 will:
- update some file and component names
- consolidate some of the components (so we don't have three operations components that don't do anything except import each other.)
- possibly do something about the routes folder

### CODE REVIEW

File: client/app/components/routes/operations/Page.tsx [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
 import { Button } from "@mui/material";
 import { Suspense } from "react";
-import Operations from "@/app/components/routes/operations/Operations";
 import Loading from "@/app/components/loading/SkeletonGrid";
 import { getServerSession } from "next-auth";
 import { authOptions } from "@/app/api/auth/[...nextauth]/route";
 import Note, { registrationRequestNote } from "../../datagrid/Note";
-import { OperationsSearchParams } from "@/app/components/routes/operations/types";
+import { OperationsSearchParams } from "../../operations/types";
+import Operations from "../../operations/Operations";
```
Comment (marcellmueller) [inline client/app/components/routes/operations/Page.tsx:10]: I think we should avoid using relative imports. Using it when it's in the same directory ie `./Operations` isn't too bad though I think going up multiple directories is confusing.  

Suggest to change all relative imports to aliased imports which are easier to read and more informative ie `@/app/components/operations/Operations`.

**Review by marcellmueller — COMMENTED** [review]
Looking good, nice work! Just one other suggestion besides the aliased imports.

While doing the `user operator` api test refactor I noticed our api now separates user operator stuff and `select operator`. Should we have a `selectOperator` directory and keep that stuff separate?

File: client/app/components/routes/operations/Page.tsx (hunk 1 above)
Comment (BCerki) [reply client/app/components/routes/operations/Page.tsx:10]: I've fixed all the multiple directories ones but left ./

**Comment by BCerki** [comment]
> Looking good, nice work! Just one other suggestion besides the aliased imports.
> 
> While doing the `user operator` api test refactor I noticed our api now separates user operator stuff and `select operator`. Should we have a `selectOperator` directory and keep that stuff separate?

I'm going to look into this in PR #2. Right now in the app folder, we've got the user operator pages within select-operator, e.g. client/app/(authenticated)/bceidbusiness/industry_user/dashboard/select-operator/(request-access)/user-operator/[id]/[formSection], so everything's mixed together. I think it probably does make sense to separate user operator and select operator in both in the app folder and in the components (there's an AC in the card to look into it).

File: client/app/components/routes/operations/Page.tsx (hunk 1 above)
Comment (marcellmueller) [reply client/app/components/routes/operations/Page.tsx:10]: I can still see about 6 imports in the changee files here that use `../../componentname`, other than that this is good to go!

File: docs/developer-guide.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Architectural design using Next.js 14 includes employing a hierarchical structur
 Next.js 14 foundation is the App Router which boasts a range of features, including:
 
 [Routing](https://nextjs.org/docs/app/building-your-application/routing)
-Creating routes involves creating folders within the app directory and nesting a page.tsx file within the appropriate folder to define your route.
+Creating routes involves creating folders within the app directory and nesting a page.tsx file within the appropriate folder to define your route. Note: our `withAuthorization` middleware removes the first three folders (e.g. authenticated/bceidbudiness/industry_user) from the route, so all the URLs within the app will start with `dashboard` instead of `authenticated/.../dashboard`.
```
Comment (marcellmueller) [inline docs/developer-guide.md:14]: Thanks for this!

**Review by marcellmueller — APPROVED** [review]
Nice job on this refactor, feels good! 🥇

**Pull request merged**